packages/search-in-workspace/src/browser/search-in-workspace-service.ts

```diff
@@ -43,12 +49,16 @@ export class SearchInWorkspaceClientImpl implements SearchInWorkspaceClient {
 
 export type SearchInWorkspaceCallbacks = SearchInWorkspaceClient;
 
+/**
+ * Service to search text in the workspace files.
+ */
+
 /**
  * Service to search text in the workspace files.
  */
 
 @injectable()
-export class SearchInWorkspaceService implements SearchInWorkspaceClient {
+export class SearchInWorkspaceService implements SearchInWorkspaceClient, ISearchInWorkspaceService {
```

Comment:
boilerplate it not encouraged: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#interfacessymbols

please inject `SearchInWorkspaceService` directly and add new methods here

Reply:
Thanks, I've removed it and now directly injecting the search service.